docs/embed-api.rst

```diff
@@ -0,0 +1,122 @@
+Embed API
```

Comment:
Why don't we put this file at `docs/api/embed.rst` instead since it's an API?

Reply:
yeah, that looks like a  better place.